src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1441,16 +1441,13 @@
      * </pre>
      * <p>parses as:</p>
      * <pre>
-     * +--CTOR_CALL (this)
-     *     |
-     *     +--LPAREN (()
-     *     +--ELIST
-     *         |
-     *         +--EXPR
-     *             |
-     *             +--NUM_INT (1)
-     *     +--RPAREN ())
-     *     +--SEMI (;)
+     * CTOR_CALL -&gt; this
+     *  |--LPAREN -&gt; (
+     *  |--ELIST -&gt; ELIST
+     *      |--EXPR -&gt; EXPR
+     *          |--NUM_INT -&gt; 1
+     *  |--RPAREN -&gt; )
+     *  |--SEMI -&gt; ;
```

Comment:
Please share output of cli for such code, I think this line should have different pseudo graphic element

Reply:
![Screenshot (535)](https://user-images.githubusercontent.com/80591258/143674822-71ef1229-eeab-479e-a8a0-b58b26b691ce.png)
Should I make it look like the AST above?
in the SUPER_CTOR_CALL example?

Reply:
Print whole ast by cli.

Reply:
@romani , I changed it, this time I created a fresh AST using CLI.
![Screenshot (539)](https://user-images.githubusercontent.com/80591258/143683864-7ee363dc-2593-47ed-a4af-b2ddacfbd10f.png)